### PR TITLE
fix typos and spelling errors/grammar

### PR DESCRIPTION
**Pull request by sinapis-amicus**
tried to fix as many spelling errors and typos as I could, some grammar too.

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -289,7 +289,7 @@ Changelog for commit "[fix(Core/Player): Use SkillLineAbility.dbc to determine p
 -    //uint32    racemaskNot;                                // 5        m_excludeRace
 -    //uint32    classmaskNot;                               // 6        m_excludeClass
 -    uint32    req_skill_value;                              // 7        m_minSkillLineRank
--    uint32    forward_spellid;                              // 8        m_supercededBySpell
+-    uint32    forward_spellid;                              // 8        m_supersededBySpell
```
Comment (Helias) [inline docs/changelog.md:292]: why you removed the comment m_* ?

File: docs/areatable.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ This DBC contains the zone and subzone lists. For the purposes of this wiki arti
 | 26       | Lighthouse                                           | 0     | 40     |
 | 28       | Western Plaguelands                                  | 0     | 0      |
 | 30       | Nine                                                 | 0     | 0      |
-| 32       | The Cemetary                                         | 0     | 10     |
+| 32       | The Cemetery                                         | 0     | 10     |
```
Comment (Kitzunu) [inline docs/areatable.md:99]: Wrong. DBC data says Cemetary.

File: docs/changelog.md [hunk 3]
```diff
@@ -302,14 +302,14 @@ Changelog for commit "[fix(Core/Player): Use SkillLineAbility.dbc to determine p
...
-+    uint32 SupercededBySpell;                               // 8
++    uint32 SupersededBySpell;                               // 8
 +    uint32 AcquireMethod;                                   // 9
 +    uint32 TrivialSkillLineRankHigh;                        // 10
 +    uint32 TrivialSkillLineRankLow;                         // 11
 +    //uint32 CharacterPoints[2];                            // 12-13
 ```
 
-- for example skillLine->forward_spellid will become skillLine->SupercededBySpell
+- for example skillLine->forward_spellid will become skillLine->SupersededBySpell
```
Comment (Kitzunu) [inline docs/changelog.md:312]: Revert these changes. They are wrong

File: docs/es/achievement_criteria_data.md [hunk 4]
```diff
@@ -53,7 +53,7 @@ Dpendiendo de este valor, será determinado cómo son usados value1 y value2.
 | 16   | TYPE_HOLIDAY              |
 | 17   | TYPE_BG_LOSS_TEAM_SCORE   |
 | 18   | TYPE_INSTANCE_SCRIPT      |
-| 19   | TYPE_S_EQUIPED_ITEM       |
+| 19   | TYPE_S_EQUIPPED_ITEM       |
```
Comment (Kitzunu) [inline docs/es/achievement_criteria_data.md:56]: Revert. This is wrong

File: docs/es/achievement_criteria_data.md [hunk 5]
```diff
@@ -143,7 +143,7 @@ Dpendiendo de este valor, será determinado cómo son usados value1 y value2.
 **TYPE\_INSTANCE\_SCRIPT**
 
 -   Hace que el script de la instancia revise que los criterios se ajusten.
-**TYPE\_S\_EQUIPED\_ITEM**
+**TYPE\_S\_EQUIPPED\_ITEM**
```
Comment (Kitzunu) [inline docs/es/achievement_criteria_data.md:146]: same here

File: docs/es/achievement_criteria_data.md [hunk 6]
```diff
@@ -183,13 +183,13 @@ Dpendiendo de este valor, será determinado cómo son usados value1 y value2.
 
 -   Max. puntaje
 
-**TYPE\_S\_EQUIPED\_ITEM**
+**TYPE\_S\_EQUIPPED\_ITEM**
```
Comment (Kitzunu) [inline docs/es/achievement_criteria_data.md:186]: same here

File: docs/es/changelog.md [hunk 7]
```diff
@@ -255,14 +255,14 @@ Registro de cambios para el commit "[fix(Core/Player): Use SkillLineAbility.dbc
 + //uint32 ExcludeRace;                                   // 5
 + //uint32 ExcludeClass;                                  // 6
 + uint32 MinSkillLineRank;                                // 7
-+ uint32 SupercededBySpell;                               // 8
++ uint32 SupersededBySpell;                               // 8
```
Comment (Kitzunu) [inline docs/es/changelog.md:258]: revert all these

File: docs/es/changelog.md [hunk 8]
```diff
@@ -255,14 +255,14 @@ Registro de cambios para el commit "[fix(Core/Player): Use SkillLineAbility.dbc
...
 + uint32 AcquireMethod;                                   // 9
 + uint32 TrivialSkillLineRankHigh;                        // 10
 + uint32 TrivialSkillLineRankLow;                         // 11
 + //uint32 CharacterPoints[2];                            // 12-13
 ```
 
-- Por ejemplo `skillLine->forward_spellid` se convertirá en `skillLine->SupercededBySpell`
+- Por ejemplo `skillLine->forward_spellid` se convertirá en `skillLine->SupersededBySpell`
```
Comment (Kitzunu) [inline docs/es/changelog.md:265]: this too

File: docs/es/changelog.md [hunk 9]
```diff
@@ -242,7 +242,7 @@ Registro de cambios para el commit "[fix(Core/Player): Use SkillLineAbility.dbc
 - //uint32    racemaskNot;                                // 5        m_excludeRace
 - //uint32    classmaskNot;                               // 6        m_excludeClass
 - uint32    req_skill_value;                              // 7        m_minSkillLineRank
-- uint32    forward_spellid;                              // 8        m_supercededBySpell
+- uint32    forward_spellid;                              // 8        m_supersededBySpell
```
Comment (Kitzunu) [inline docs/es/changelog.md:245]: this too

File: docs/achievement_criteria_data.md [hunk 10]
```diff
@@ -53,7 +53,7 @@ Depending on this value, it will determine how value1 and value2 are used.
 | 16   | TYPE_HOLIDAY              |
 | 17   | TYPE_BG_LOSS_TEAM_SCORE   |
 | 18   | TYPE_INSTANCE_SCRIPT      |
-| 19   | TYPE_S_EQUIPED_ITEM       |
+| 19   | TYPE_S_EQUIPPED_ITEM       |
```
Comment (Kitzunu) [inline docs/achievement_criteria_data.md:56]: revert

File: docs/achievement_criteria_data.md [hunk 11]
```diff
@@ -144,7 +144,7 @@ Depending on this value, it will determine how value1 and value2 are used.
 
 -   Make instance script call for check current criteria requirements fit.
 
-**TYPE\_S\_EQUIPED\_ITEM**
+**TYPE\_S\_EQUIPPED\_ITEM**
```
Comment (Kitzunu) [inline docs/achievement_criteria_data.md:147]: revert

File: docs/achievement_criteria_data.md [hunk 12]
```diff
@@ -184,13 +184,13 @@ Depending on this value, it will determine how value1 and value2 are used.
 
 -   Max. score
 
-**TYPE\_S\_EQUIPED\_ITEM**
+**TYPE\_S\_EQUIPPED\_ITEM**
```
Comment (Kitzunu) [inline docs/achievement_criteria_data.md:187]: revert

File: docs/changelog.md [hunk 13]
```diff
@@ -302,14 +302,14 @@ Changelog for commit "[fix(Core/Player): Use SkillLineAbility.dbc to determine p
 +    //uint32 ExcludeRace;                                   // 5
 +    //uint32 ExcludeClass;                                  // 6
 +    uint32 MinSkillLineRank;                                // 7
-+    uint32 SupercededBySpell;                               // 8
++    uint32 SupersededBySpell;                               // 8
```
Comment (Kitzunu) [inline docs/changelog.md:305]: revert

File: docs/changelog.md (hunk 1 above)
Comment (Kitzunu) [inline docs/changelog.md:292]: revert

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: docs/changelog.md (hunk 1 above)
Comment (sinapis-amicus) [reply docs/changelog.md:292]: I did not remove the comment. Spelling was corrected to m_supersededBySpell from m_supercededBySpell

Comment (Kitzunu) [reply docs/changelog.md:292]: It wasnt removed, the C was changed to S. But that is wrong, as in the core it is defined with C

Comment (Kitzunu) [reply docs/changelog.md:292]: While these changes are appreciated, the variables and stuff needs to be checked with how they are defined in the core before changes are made.

**Review by Kitzunu — CHANGES_REQUESTED** [review]
still stuff to revert =)

**Review by sinapis-amicus — COMMENTED** [review]
requested changes should be taken care of

**Review by Helias — APPROVED** [review]
